dynatrace-oneagent-operator/README.md

```diff
@@ -69,6 +90,23 @@ You can then check for the latest version by searching your Helm repositories fo
 $ helm search repo dynatrace-oneagent-operator
 ```
 
+Download the latest version of the CRD attached to [the latest release](https://github.com/Dynatrace/dynatrace-operator/releases) and apply it
```

Comment:
also here

Reply:
Updated the links and the commands accordingly